src/openms/source/APPLICATIONS/TOPPBase.cpp

```diff
@@ -169,6 +169,7 @@ namespace OpenMS
     registerStringOption_("write_ctd", "<out_dir>", "", "Writes the common tool description file(s) (Toolname(s).ctd) to <out_dir>", false, true);
     registerStringOption_("write_wsdl", "<file>", "", "Writes the default WSDL file", false, true);
     registerFlag_("no_progress", "Disables progress logging to command line", true);
+    registerFlag_("force", "Overwrite tool specific checks.", true);
```

Comment:
hmm would not mention it explicitly... how it is used is tool dependent. Adding a documentation here is too easily missed if someone uses it in a tool for different purposes.